target/linux/ath79/image/nand.mk

```diff
@@ -46,6 +46,22 @@ define Device/aerohive_hiveap-121
 endef
 TARGET_DEVICES += aerohive_hiveap-121
 
+define Device/domywifi_dw33d
+  SOC := qca9558
+  DEVICE_VENDOR := DomyWifi
+  DEVICE_MODEL := DW33D
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-storage kmod-usb-ledtrig-usbport kmod-ath10k-ct ath10k-firmware-qca988x-ct
```

Comment:
One could introduce a line break here ...

Reply:
Oh. I overlooked this. Fixed in my staging tree. Thanks for spotting it.

BTW It's weird that checkpatch.pl doesn't report this problem.

Reply:
I reorganized the code and submission information, and modified the signature. But the main line upstream code seems to have a problem, do not compile ar934x_nand drivers, in addition, the firmware is operating normally.